gems/pending/util/vmdb-logger.rb

```diff
@@ -101,8 +120,30 @@ def log_hashes(h, options = {})
     self.class.log_hashes(self, h, options)
   end
 
+  def level
+    local_level || super
+  end
+
   private
 
+  def local_log_id
+    Thread.current.object_id
+  end
+
+  def local_level
+    @local_levels[local_log_id]
+  end
+
+  def local_level=(level)
+    @write_lock.syncronize do
```

Comment:
Huh, I basically did the "QA Steps" in order as my way of working through this myself, and didn't have a problem with that, so I am confused why I didn't get an error.  Let me see if that is aliased, but yeah, otherwise I agree.